Source/Core/UICommon/ResourcePack/ResourcePack.cpp

```diff
@@ -19,6 +21,7 @@
 namespace ResourcePack
 {
 constexpr char TEXTURE_PATH[] = "Load/Textures/";
+constexpr char DYNAMIC_INPUT_PATH[] = "Load/DynamicInputTextures";
```

Comment:
`TEXTURE_PATH` has a trailing slash, while this one does not. Does it make a difference to use either for consistency? I for one prefer folder paths to end in a slash so I don't have to worry about it elsewhere.